core/templates/pages/teach-page/teach-page.component.ts

```diff
@@ -16,64 +16,62 @@
  * @fileoverview Component for the teach page.
  */
 
-require('base-components/base-content.directive.ts');
-require(
-  'components/common-layout-directives/common-elements/' +
-  'background-banner.component.ts');
+import { Component, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
 
-require('domain/utilities/url-interpolation.service.ts');
-require('services/site-analytics.service.ts');
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+import { SiteAnalyticsService } from 'services/site-analytics.service';
+import { WindowRef } from 'services/contextual/window-ref.service';
```

Comment:
This is still not sorted according to path. This should move above.

Reply:
Done